javascript/widgets/handler/callback.js

```diff
@@ -92,6 +92,19 @@ firebaseui.auth.widget.handler.handleCallback =
         firebaseui.auth.widget.handler.handleCallbackFailure_(
             app, component, /** @type {!Error} */ (error));
       }
+    } else if (error &&
+        error['code'] == 'auth/operation-not-supported-in-this-environment' &&
+        firebaseui.auth.widget.handler.common.isPasswordProviderOnly(app)) {
```

Comment:
There are cases where the app enables password only but previously the user had created an account with Google and then tries to sign in with the same email. I prefer this conservative approach as the configuration providers is not a guarantee on the method of sign in.